experiments/dgps.R

```diff
@@ -0,0 +1,125 @@
+# dgps.R - convenience script for generating simulation data for grf.
```

Comment:
Could you please create a new directory experiments/benchmarking, and put this in there? I don't think we want this to be at the top level of the experiments directory.